.openshift-ci/crawler/env.sh

```diff
@@ -18,6 +18,12 @@ gcloud secrets versions access latest \
 
 echo >> $BASH_ENV
 
+gcloud secrets versions access latest \
+    --secret="collector-kernel-crawler-suse-credentials" \
+    | sed -e 's/^/export /' >> $BASH_ENV
+
```

Comment:
Seems like this code duplication should be refactored with the use of a function.